.cbmc-batch/include/proof_helpers/utils.h

```diff
@@ -134,3 +134,8 @@ uint64_t nondet_hasher(const void *a);
  * Standard stub function to hash one item.
  */
 uint64_t uninterpreted_hasher(const void *a);
+
+/**
+ * Standard stub function of a predicate
+ */
+bool __CPROVER_uninterpreted_predicate_fn(uint8_t value);
```

Comment:
Please don't use the `__CPROVER` prefix, those are for CBMC to use only. I think `uninterpreted_predicate_fn` is fine, or you could invent some other prefix if you'd like to make it clear that these are being used in CBMC proofs.